.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+  "printWidth": 100,
+  "tabWidth": 2,
+  "singleQuote": true,
+  "trailingComma": "all",
+  "bracketSpacing": true,
+  "jsxBracketSameLine": true,
```

Comment:
If we will be using Prettier `>= 2.4`, this property is just called `bracketSameLine` now instead of `jsxBracketSameLine`
https://prettier.io/blog/2021/09/09/2.4.0.html